### PR TITLE
feat(multi): multiple field join

### PR DESCRIPTION
**Pull request by thanh-nguyen-dang**
Jira Ticket: [PXP-8509](https://ctds-planx.atlassian.net/browse/PXP-8509)

### New Features
Multiple field for joining two indices

### CODE REVIEW

File: tube/etl/indexers/aggregation/translator.py [hunk 1]
```diff
@@ -256,34 +288,46 @@ def join_to_an_index(self, df, translator, joining_node):
...
         # field which is identity of a node is named as _{node}_id now
         # before in etl-mapping for joining_props, we use {node}_id
         # for backward compatibility, we check first with the value in mapping file.
         # if there is not any Prop object like that, we check with new format _{node}_id
-        id_field_in_df = self.parser.get_prop_by_name(joining_node.joining_field)
-        if id_field_in_df is None:
+        id_fields_in_df = [
+            self.parser.get_prop_by_name(f) for f in joining_node.joining_fields
+        ]
+        if id_fields_in_df is None:
```
Comment (m0nhawk) [inline tube/etl/indexers/aggregation/translator.py:302]: I'm not sure this is correct comparison, `id_fields_in_df` cannot be `None` now, and will at most be an empty array.

```
>>> print([] is None)
False
```

File: tube/etl/indexers/aggregation/translator.py [hunk 2]
```diff
@@ -256,34 +288,46 @@ def join_to_an_index(self, df, translator, joining_node):
...
             id_field_in_df = self.parser.get_prop_by_name(
```
Comment (m0nhawk) [inline tube/etl/indexers/aggregation/translator.py:303]: Again, not sure if it's a correct variable... Isn't id should be `id_fields_in_df`?

File: tube/etl/indexers/aggregation/translator.py (hunk 1 above)
Comment (thanh-nguyen-dang) [reply tube/etl/indexers/aggregation/translator.py:302]: thanks. That is correct

Comment (thanh-nguyen-dang) [reply tube/etl/indexers/aggregation/translator.py:302]: thanks. that is correct

File: tube/etl/indexers/aggregation/translator.py (hunk 2 above)
Comment (thanh-nguyen-dang) [reply tube/etl/indexers/aggregation/translator.py:303]: yeah, I corrected it. thanks

**Review by m0nhawk — APPROVED** [review]

**Pull request merged**